docs/source/mistral_yaql.rst

```diff
@@ -0,0 +1,201 @@
+Mistral + YAQL
+==============
+YAQL is typically used for simple conditional evaulation and data transformation in Mistral 
+workflows. There will be many cases where you did not author the actions but there's a need to 
+decide from the result of the action whether to continue or there's a need to transform the 
+result to another value or structure for the next action in the workflow.
+
+Here are some examples of usages.
+
+* Select values for a key from a list of dictionary.
```

Comment:
I think the usage is "what for",  like "define conditions on transitions between tasks", "pass input from upward task results" and "form values for publish data or workflow output". This list is good highlight of YAQL power features. "It can do cool stuff like". 
Make somethign right here what YAQL can do and Jinga can or suck at, will be good, too.